day_6/exercises/burrito.js

```diff
@@ -16,11 +16,28 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+
+  addTopping(newTopping) {
+    this.toppings.push(newTopping);
+  }
+
+  removeTopping(noTopping) {
```

Comment:
What would be a better name for this var? Keep in mind verbosity and clarity are best when naming things